### PR TITLE
Improve the documentations

### PR DESCRIPTION
**Pull request by nikomakela**
HCRC-82

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,13 +88,18 @@ We provide utilities for fetching headless data for NextJs in `react-helsinki-he
...
 | `yarn dev`            | Starts storybook environment that can be used for developing components.                                                   |                               |
 | `yarn lint`           | Lints the application to be according to quality standards (eslint) and formatting standards (prettier).                   | `--fix`: fix fixable problems |
 | `yarn test`           | Runs tests with jest.                                                                                                      | `--watch`: enable watch mode  |
 | `yarn build`          | Builds application with rollup.                                                                                            |                               |
 | `yarn publish-canary` | Publishes a canary tagged version of the application. CD is configured to run this script on additions tot he main branch. |                               |
+| `yarn publish-stable` | Publishes a stable tagged version of the application. CD is configured to run this script on additions tot he main branch. |                               |
```
Comment (karisal-anders) [inline README.md:100]: typo: tot he -> to the

File: README.md [hunk 2]
```diff
@@ -118,12 +125,35 @@ On a new release, a new version is released to npm.
 
 ### Storybook
 
+Storybook is a frontend workshop for building UI components and pages in isolation https://storybook.js.org/. The Storybook can be used to develop and to test the components, but also to documentate the components and their features.
```
Comment (karisal-anders) [inline README.md:128]: documentate -> document (i.e. documentate is not a word, document is a valid verb)

**Review by karisal-anders — APPROVED** [review]
A quick glance, can't really evaluate the storybook changes with my current level of frontend expertise without taking more time to look into them and didn't take that at the moment. A couple of typo/word choices in README. Looks ok to merge!

Please if someone else could take a look at the storybook changes? Thank you!

**Comment by nikomakela** [comment]
> A quick glance, can't really evaluate the storybook changes with my current level of frontend expertise without taking more time to look into them and didn't take that at the moment. A couple of typo/word choices in README. Looks ok to merge!
> 
> Please if someone else could take a look at the storybook changes? Thank you!

The main thing at this point is to share some examples and give a hint what could be done. I think it responds on that. I'm sure it's not perfect yet and I even noticed some inconsistencies while I went through them, but I don't know if it really pays back to refactor those things now. Let's do that later when we have some unit tests we can rely to.

**Pull request merged**